### PR TITLE
youtube data api hook basic example

### PR DESCRIPTION
**Pull request by andreinechaev**

### CODE REVIEW

File: mediamanager/objects/encoder.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import json
+import datetime
+
+class ObjectEncoder(json.JSONEncoder):
```
Comment (eIGato) [inline mediamanager/objects/encoder.py:4]: Please use 2 empty lines before any class definition.

File: mediamanager/objects/video.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        return {
+            'id': self.id,
+            'videoId': self.videoId,
+            'channelId': self.channelId,
+            'title': self.title,
+            'link': self.link,
+            'author': self.author,
+            'uri': self.uri,
+            'published': self.published,
+            'updated': self.updated
```
Comment (eIGato) [inline mediamanager/objects/video.py:41]: Add a trailing comma.

File: webapp/backend/tests/hooks_test.py [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                    <author>
+                    <name>Channel title</name>
+                    <uri>http://www.youtube.com/channel/CHANNEL_ID</uri>
+                    </author>
+                    <published>2015-03-06T21:40:57+00:00</published>
+                    <updated>2015-03-09T19:05:24.552394234+00:00</updated>
+                </entry>
+                </feed>'''
+
+@pytest.fixture(scope='module')
```
Comment (eIGato) [inline webapp/backend/tests/hooks_test.py:26]: 2 empty lines.

File: webapp/backend/tests/hooks_test.py [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+def celery_app(request):
+    celery.conf.update(task_always_eager=True)
+    return celery
+
+@pytest.fixture()
```
Comment (eIGato) [inline webapp/backend/tests/hooks_test.py:31]: 2 empty lines.

File: mediamanager/objects/video.py (hunk 2 above)
Comment (andreinechaev) [reply mediamanager/objects/video.py:41]: it is not a common practice, rather can be a team definition. Everything that you mentioned should be taken by a hook on commit with autopep or similar. Trailing coma is not a definition of PEP

**Pull request merged**